File: LeetCode Solution(Python)/Length-of-Last-Word.py
# Solution to problem 58: Length of Last Word
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        s = s.strip()
        if not s:
            return 0
        return len(s.split()[-1])

File: LeetCode Solution(CPP)/Length-of-Last-Word.cpp
#include<iostream>
#include<vector>
using namespace std;

class Solution {
public:
    int lengthOfLastWord(string s) {
        //soltuion to problem 58, length of last word
        int length = 0;
        for (int i = s.size() - 1; i >= 0; i--) {
            if (s[i] != ' ') {
                length++;
            } else if (length > 0) {
                break;
            }
        }
        return length;
    }
};